test-data/unit/fine-grained.test

```diff
@@ -996,3 +996,232 @@ class B:
 ==
 ==
 a.py:4: error: Revealed type is 'builtins.int'
+
+[case testStripRevealType]
+import a
+reveal_type(a.f())
+[file a.py]
+def f() -> int: pass
+[file a.py.2]
+def f() -> str: pass
+[out]
+main:2: error: Revealed type is 'builtins.int'
+==
+main:2: error: Revealed type is 'builtins.str'
+
+[case testDecoratorTypeAfterReprocessing]
+import a
+reveal_type(a.f())
+[file a.py]
+from contextlib import contextmanager
+from typing import Iterator
+import b
+@contextmanager
+def f() -> Iterator[None]:
+    yield
+[file b.py]
+[delete b.py.2]
+[file b.py.3]
+[typing fixtures/typing-full.pyi]
+[builtins fixtures/list.pyi]
+[triggered]
+2: <b>
+3: <b>
+[out]
+main:2: error: Revealed type is 'contextlib.GeneratorContextManager[builtins.None]'
+==
+a.py:1: error: Cannot find module named 'b'
+a.py:1: note: (Perhaps setting MYPYPATH or using the "--ignore-missing-imports" flag would help)
+main:2: error: Revealed type is 'contextlib.GeneratorContextManager[builtins.None]'
+a.py:3: error: Cannot find module named 'b'
```

Comment:
There's only one "import b" in a.py (on line 3) -- why does that error get reported both for line 1 and for line 3?

Reply:
That's a bug with the current implementation. There's a TODO comment elsewhere but I haven't repeated it everywhere where this happens.